manifests/f/focus@will/focusatwill/1.6.0.44/focus@will.focusatwill.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: focus@will.focusatwill
+PackageVersion: 1.6.0.44
+PackageLocale: en-US
+Publisher: focus@will
+PackageName: focusatwill
+License: Copyright © 2019 focus@will

```

Comment:
A copyright is no a license; Please provide the actual package license terms such as `Proprietary`, `MIT`, `GPL-2.0` or whatever package license terms are applicable

Reply:
ok, so... I don't know what that means... 😬